.github/workflows/ci.yml

```
@@ -26,17 +26,17 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rails_version: [5.2.5, 6.0.3.6, 6.1.3.1, main]
-        ruby_version: [2.5, 2.6, 2.7, 3.0]
+        rails_version: ['5.2.6', '6.0.4.4', '6.1.4.4', main]
```

```suggestion
        rails_version: ['5.2.6', '6.0.4.4', '6.1.4.4', 'main']
```

May want to quote this for consistency. This whole debacle seems to have been a nice little reminder that we can't rely on YAML's implicit typing. 😅 Now I see why folks aren't so keen on it...